lib/tasks/index.rake

```diff
@@ -14,13 +14,15 @@ namespace :index do
   task dspace_research_data: :environment do
     Rails.logger.info "Harvesting and indexing DataSpace research data collections started"
     DspaceResearchDataHarvester.harvest(false)
+    Blacklight.default_index.connection.commit
```

Comment:
💯 for identifying this, although it concerns me that `autocommit` does not automatically address this.

Reply:
Autocommit probably does the right thing, this was more of a belt and suspenders kind of thing since I was grasping at straws at times.